### DISCUSSION TITLE
Hubs Cloud - ShortlinkZoneSSLCertEast still in process until rollback

### DISCUSSION THREAD

**Discussion by fredNeko** [mozilla/hubs#2695] [General Questions & Discussion] [answered]
Hello Everyone,

I'm trying to create a stack on AWS but one part of the procedure still in process (ShortlinkZoneSSLCertEast) until the rollback procedure.
I'm really neebie in AWS and cloud tech in general but i don't know where I can fix that.
For info, I'm using servers on Ireland, and I already created a domain name in Route 52.
Can you help me ?

Thx in advance.

**Comment by robinkwilson** [comment, marked as answer]
Hello! Awesome thank you for the information. 

Quick first question: 
- Did you follow the [Domain Recipe Guide in our docs](https://hubs.mozilla.com/docs/hubs-cloud-aws-domain-recipes.html)? 

You need at least 2 domains to setup the stack: 
1. Main and internal domains can be on the first domain 
2. Shortlink domain needs to be on the second domain

Please let me know!

**Comment by fredNeko** [comment]
Hello Robin,

Your answer is resolving the issue for the shortlinkZoneSSLCertEast but now i have a similar issue with the ExternalZoneSSLCertEast.
Now, I have 2 domains one in .com and the other in .link. Both are dedicated in Route 52.
I see the SSL certificate creations in Certificate Manager but in Virginia North the _mydomain.com still pending.
All entries in the domains DNS seams to be done correctly.

Any ideas ?
Thx again for your time.
![Capture7](https://user-images.githubusercontent.com/57808979/88344142-41e12b00-cd43-11ea-87d5-ec5a0fcb1455.JPG)

**Comment by robinkwilson** [comment]
Did your domains leave the "Pending Requests" menu (aws console > Route 53 > Pending Requests) to finish registering? Try the stack deploy again. 

![image](https://user-images.githubusercontent.com/29560735/88345643-408c1000-ccfb-11ea-93e7-345bf0584917.png)

**Reply by fredNeko** [reply]
No pending requests. 
I waited until the link domain procedure was done.

I tried 2 times to deploy the stack with the same issue.

Do you think I've reach the maximum number of SSL  / month.
In your documentation, you are mentionning 25 certificates / month
Were canI check that ?

Thx

**Comment by fredNeko** [comment]
Hello,

I tried to deploy a new stack this morning and It's working now
 I think it was just a question of timing to spread the domain on the net because I didn't change anything ! 

@robin-k-wilson Thanks for your support to solve my issue.
